docs/assets/zdm-config-reference.yml

```diff
@@ -0,0 +1,164 @@
+# This variable determines which cluster is currently considered the primary cluster.
+# At the start of the migration, the primary cluster is Origin, as it contains all the data.
+# In Phase 4 of the migration, once all the existing data has been transferred and any validation/reconciliation
+# step has been successfully executed, you can switch the primary cluster to be Target.
+# Valid values: ORIGIN, TARGET.
+primary_cluster: ORIGIN
+
+# This variable determines how reads are handled by the ZDM Proxy. Valid values:
+# PRIMARY_ONLY - reads are only sent synchronously to the primary cluster. This is the default behavior.
+# DUAL_ASYNC_ON_SECONDARY - reads are sent synchronously to the primary cluster and also asynchronously
+# to the secondary cluster. See Phase 3: Enable asynchronous dual reads.
+read_mode: PRIMARY_ONLY
+
+# Whether the ZDM Proxy should replace standard CQL function calls in write
+# requests with a value computed at proxy level. Currently, only the replacement
+# of now() is supported. Disabled by default. Enabling this will have a noticeable performance impact.
+replace_cql_functions: false
+
+# Timeout (in ms) when performing the initialization (handshake) of a proxy-to-secondary cluster
+# connection that will be used solely for asynchronous dual reads. If this timeout occurs, the asynchronous
+# reads will not be sent. This has no impact on the handling of synchronous requests: the ZDM Proxy will
+# continue to handle all synchronous reads and writes normally.
+async_handshake_timeout_ms: 4000
+
+# Specifies logging level.
+log_level: INFO
+
+# TODO: ProxyTopologyIndex
+# TODO: ProxyTopologyAddresses
+# TODO: ProxyTopologyNumTokens
+
+# Comma separated list of origin cluster contact points.
+origin_contact_points: origin.datastax.com
+
+# Port used when connecting to nodes from origin cluster.
+origin_port: 9042
+
+# If origin cluster is DataStax Astra, path to secure connection bundle.
+# origin_secure_connect_bundle_path:
+
+# Local data center for origin cluster.
+# origin_local_datacenter:
+
+# Origin cluster username.
+origin_username: user1
+
+# Origin cluster password.
+origin_password: pass1
+
+# Timeout (in ms) when attempting to establish a connection from the proxy to origin cluster.
+origin_connection_timeout_ms: 30000
+
+# CA certificate used when verifying identity of origin nodes.
+# origin_tls_server_ca_path:
+
+# Public key used when establishing connectivity with origin cluster.
+# origin_tls_client_cert_path:
+
+# Private key used to secure communication with origin cluster.
+# origin_tls_client_key_path:
+
+# Comma separated ist of target cluster contact points.
+target_contact_points: target.datastax.com
```

Comment:
maybe 127.0.0.2 as target and 127.0.0.1 as origin? feels a bit weird to see a valid hostname here (even if it doesn't exist, at least I don't think it does? but the datastax.com domain definitely exists 😅 )

Reply:
Good point, fixed.